src/com/b44t/messenger/rpc/RpcError.java

```diff
@@ -0,0 +1,11 @@
+package com.b44t.messenger.rpc;
+
+/**
+ * An exception occurred while processing a request.
+ **/
+public class RpcError extends Throwable {
```

Comment:
You should extend Exception or RuntimeException here, not Throwable, because to catch "everything catchable" we often write `catch (Exception e)`. Errors that extend Throwable but not Exception are meant to not be catched at all, like an OutOfMemoryError (idk why it's not meant to be catched, to me it seems like recovering is worth a try, but anyway).

Might also call it `RpcException` instead of `RpcError`, because in Java Errors are usually something unrecoverable, but that's more a matter of taste.

Or we might even get rid of RpcError completely, but then we need some other way of handling errors, prob. returning an object that either contains the result or the error.